docker-ubuntu/Dockerfile

```diff
@@ -0,0 +1,104 @@
+#
+# The docker image is built in two stages.  First stage builds the
+# oasis-node and oasis-core-rosetta-gateway binaries.
+# Second stage prepares the execution environment and copies the
+# two binaries over.
+#
+
+#
+# Build stage
+#
+FROM golang:1.16-bullseye AS build
+
+# Install prerequisites.
+RUN apt-get update && apt-get install -y bzip2 libseccomp-dev
+
+ARG CORE_BRANCH=v21.3.5
```

Comment:
Should we bump to 21.3.9?

Reply:
ya let's do that for both

Reply:
oh nvm let's try replacing the original, so we'll bump just this one

Reply:
ok bumped, along with makefile and go module. was going to bump the ed25519 library too correspondingly. we actually changed to a different library in oasis-core. but I changed it no longer to need a dedicated ed25519 library